aio/content/examples/schematics-for-libraries/projects/my-lib/schematics/my-service/index.ts

```diff
@@ -6,37 +6,51 @@ import {
   chain, mergeWith
 } from '@angular-devkit/schematics';
 
-import { strings, normalize, experimental } from '@angular-devkit/core';
+import { strings, normalize, virtualFs, workspaces } from '@angular-devkit/core';
 // #enddocregion schematics-imports
 
 import { Schema as MyServiceSchema } from './schema';
 // #enddocregion schema-imports
 
-export function myService(options: MyServiceSchema): Rule {
-  return (tree: Tree) => {
-    const workspaceConfig = tree.read('/angular.json');
-    if (!workspaceConfig) {
-      throw new SchematicsException('Could not find Angular workspace configuration');
-    }
+export function createHost(tree: Tree): workspaces.WorkspaceHost {
```

Comment:
The `export` is redundant here.

```suggestion
function createHost(tree: Tree): workspaces.WorkspaceHost {
```